ddl/stage_ingest_index.go

```diff
@@ -61,9 +61,9 @@ func (i *ingestIndexStage) Init(ctx context.Context) error {
 	return err
 }
 
-func (*ingestIndexStage) SplitSubtask(ctx context.Context, _ *proto.Subtask) ([]proto.MinimalTask, error) {
```

Comment:
Maybe we can move code in init function to RunSubtask

Reply:
init is done once for all subtasks, RunSubtask is per subtask